config.schema.json

```diff
@@ -75,6 +75,25 @@
             }
           }
         }
+      },
+      "zones": {
+        "title": "Zone cleaning",
+        "type": "array",
+        "required": false,
+        "items": {
+          "title": "Zones",
+          "type": "object",
+          "properties": {
+            "name": {
+              "name": "Name",
+              "type": "string"
+            },
+            "zone": {
+              "name": "Zoon",
```

Comment:
Zoon? Is that German? :)

Reply:
Typo ... Zone ... 

I will update both...